evm_arithmetization/src/generation/linked_list.rs

```diff
@@ -42,6 +44,20 @@ where
     _marker: PhantomData<T>,
 }
 
+// Provides quick access to pointers that reference the memory location
+// of a storage or accounts linked list node, containing a specific key.
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+pub(crate) struct LinkedListPtrs {
+    /// Each entry contains the pair (key, ptr) where key is the (hashed) key
+    /// of an account in the accounts linked list, and ptr is the respective
+    /// node address in memory.
+    pub(crate) accounts_ptrs: BTreeMap<U256, usize>,
+    /// Each entry contains the pair ((account_key, slot_key), ptr) where
+    /// account_key is the (hashed) key of an account, slot_key is the slot
+    /// key, and ptr is the respective node address in memory.
+    pub(crate) storage_ptrs: BTreeMap<(U256, U256), usize>,
```

Comment:
pretty nitpicky, but the type (`LinkedListPtrs`) already carries the info that these are pointers, and the struct field too (`state_ptrs`), so I would simplify the namings here to simply `accounts` and `storage`, so that below we have for instance `state_ptrs.accounts`.
But personal preference, non-blocking.